credentialdigger/client.py

```diff
@@ -874,6 +917,8 @@ def _scan(self, repo_url, scanner, models=None, force=False, debug=False,
             except ModuleNotFoundError:
                 logger.warning('SnippetModel not found. Skip it.')
 
+        if similarity:
```

Comment:
if it takes some time, please add a log. Something like
`logger.info('Build embedding model for this repository')`
otherwise the scan will look like pending